### ISSUE TITLE
Save&Quit

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#716]
would be nice to have that option on dailies but it's prolly rly hard to implement i imagine

**Comment by Lonrite** [comment]
also there should be a little animation with isaac running next to 0 (like during loading) if the person is currently doing the daily, and only 0 without anything if they alt+f4ed or crashed

**Comment by ZMBanshee** [comment]
I'm guessing the reason this doesn't exist is because it would be abused beyond belief.

**Comment by Lonrite** [comment]
yeah i thought about it too, but then again you can already submit any daily you want anyway even after the day ended
that's why i didn't mention that
